cmd/sid/cli/btc_headers.go

```diff
@@ -0,0 +1,139 @@
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	babylontypes "github.com/babylonchain/babylon/types"
+	bbnbtclightclienttypes "github.com/babylonchain/babylon/x/btclightclient/types"
+	"github.com/babylonchain/staking-indexer/btcclient"
+	"github.com/babylonchain/staking-indexer/config"
+	"github.com/babylonchain/staking-indexer/log"
+	"github.com/babylonchain/staking-indexer/utils"
+	"github.com/urfave/cli"
+	"go.uber.org/zap"
+
+	sdkmath "cosmossdk.io/math"
+)
+
+const (
+	outputFileFlag = "outputfile-path"
+)
+
+var BtcHeaderCommand = cli.Command{
+	Name:        "btc-headers",
+	Usage:       "btc-headers 10 15 --outputfile-path ~/myoutput/path/btc-headers.json",
```

Comment:
```suggestion
	Usage:       "btc-headers [from] [to] [--output=path/to/btc-headers.json]",
```

Looks like an example, not usage

Reply:
https://github.com/babylonchain/staking-indexer/pull/93/commits/fab2bf6439951c022dc46a08c5a1968e0777c8d4